ca/proto/ca.proto

```diff
@@ -21,18 +21,24 @@ message IssueCertificateRequest {
 
   // certProfileName is a human readable name provided by the RA and used to
   // determine if the CA can issue for that profile. A default name will be
-  // assigned during *Profile construction if no name is provided.
+  // assigned inside the CA during *Profile construction if no name is provided.
+  // The value of this field should not be relied upon inside the RA.
   string certProfileName = 5;
 }
 
 message IssuePrecertificateResponse {
-  // Next unused field number: 3
+  // Next unused field number: 4
   bytes DER = 1;
 
   // certProfileHash is a hash over the exported fields of a certificate profile
   // to ensure that the profile remains unchanged after multiple roundtrips
   // through the RA and CA.
   bytes certProfileHash = 2;
+
+  // certProfileName is a human readable name returned back to the RA for later
+  // use. If the reciprocal field in IssueCertificateRequest was an empty
```

Comment:
```suggestion
  // use. If the IssueCertificateRequest.certProfileName was an empty
```

I had a moment of confusion reading reciprocal, since I read it in the mathematical sense, which metaphorically I understand as "opposite." At any rate we can be clearer by just naming the field. :D